### ISSUE TITLE
How do you ensure that the comparisons are fair?

### ISSUE THREAD

**Issue by huangshenno1** [embeddings-benchmark/mteb#105]
Hello, this work is wonderful. However, I have one question: How do you ensure that the comparisons are fair? Data leak may occur if some of the models use the train/test data for pretraining or finetuning, particularly for new models to submit.

**Comment by Muennighoff** [comment]
Yeah, we mention that in the [paper](https://arxiv.org/abs/2210.07316), where for Reranking MPNet & MiniLM perform very well likely due to SciDocsRR being part of their training data.

However, in the limit, i.e. the more datasets are benchmarked, this effect becomes negligible. So our solution is to just benchmark on a lot of datasets & samples.

Even if one were to adversarially train a model on all benchmarked test datasets, I think it would be hard to achieve a great score due to the variety of datasets & evaluation regimes.

**Comment by KennethEnevoldsen** [comment]
@huangshenno1 seems like this answers your question I will close this issue for now.
